src/ui/public/styles/base.less

```diff
@@ -54,6 +54,10 @@ code {
   word-wrap: break-word;
 }
 
+main {
+  display: block;
+}
```

Comment:
Can we move this to https://github.com/elastic/kibana/blob/master/ui_framework/components/_common_styles.scss and add a comment?

```scss
/**
 * 1. Required for IE11.
 */
main {
  display: block; /* 1 */
}
```